apps/src/code-studio/assets/SpriteManagementDirectory.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export default class SpriteManagementDirectory extends React.Component {
```

Comment:
nit: is Directory the right name? Maybe Index?

Reply:
I'm tempted to leave it as directory, unless there is a specific reason to change it. Mostly because there are fewer instances of "directory" than "index" so it's easier to search and find the relevant bits.